docs/src/pages/components/snackbars/ConsecutiveSnackbars.tsx

```diff
@@ -1,111 +1,98 @@
-import React, { SyntheticEvent } from 'react';
-import { createStyles, withStyles, WithStyles, Theme } from '@material-ui/core/styles';
+import React, { useRef, SyntheticEvent } from 'react';
+import { makeStyles, Theme } from '@material-ui/core/styles';
 import Button from '@material-ui/core/Button';
 import Snackbar from '@material-ui/core/Snackbar';
 import IconButton from '@material-ui/core/IconButton';
 import CloseIcon from '@material-ui/icons/Close';
 
-const styles = (theme: Theme) =>
-  createStyles({
-    close: {
-      padding: theme.spacing(0.5),
-    },
-  });
+const useStyles = makeStyles((theme: Theme) => ({
+  close: {
```

Comment:
We still need createStyles here.

Reply:
Should I use `createStyles` in replace of `makeStyles`?

Reply:
Sorry, I was a bit vague. Here's an example of what I mean: https://github.com/mui-org/material-ui/blob/7bac6c53485eb8ddfafe0c0b0c233d46166cde79/docs/src/pages/components/breadcrumbs/RouterBreadcrumbs.tsx#L45-L60